### DISCUSSION TITLE
MODULE_NOT_FOUND

### DISCUSSION THREAD

**Discussion by bagushermawan** [ArugaZ/whatsapp-bot#468] [Q&A] [answered]
error setelah npm start, kenapa ya bang

> whatsapp-bot@3.0.0 start
> node ./dist/main.js

node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module 'D:\zzz\botstickerwa\dist\main.js'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

![image](https://github.com/arugaz/whatsapp-bot/assets/54248013/a92b7df4-e72e-408f-9c79-04d5decdd137)

**Comment by arugaz** [comment]
harus di build dlu

**Reply by bagushermawan** [reply]
jadi gimana tu bang?

**Reply by arugaz** [reply, marked as answer]
```
npm run build
```
setelah itu
```
npm start
```
